src/components/PrimaryPanes/Sources.css

```diff
@@ -70,6 +74,10 @@
   display: inline-block;
 }
 
+.sources-list .tree .tree-node:not(.focused):hover {
+  background-color: #204e8a;
```

Comment:
is there a good variable?

also should we do it in core?
https://github.com/devtools-html/devtools-core/issues/912

Reply:
Yes, fixed